test/test_jit.py

```diff
@@ -7880,9 +7846,15 @@ def func(a):
             return c
 
         graph = torch.jit.script(func).graph
+        self.assertTrue("int = prim::Constant" in str(graph))
```

Comment:
hmm the same comment as above, why we are having both assertTrue and FileCheck, it would be better to keep a single style if possible